src/mechanics/timeIntegration/NewmarkBase.h

```diff
@@ -29,61 +29,16 @@ class NewmarkBase : public TimeIntegrationBase
     	mMuDampingMass = rMuDampingMass;
     }
 
-    double GetDampingCoefficientMass()const
-    {
-    	return mMuDampingMass;
-    }
-
     void SetToleranceForce(double rToleranceForce)
     {
     	mToleranceForce = rToleranceForce;
     }
 
-    double GetToleranceForce()const
-    {
-    	return mToleranceForce;
-    }
-
     void SetMaxNumIterations(int rMaxNumIterations)
     {
     	mMaxNumIterations = rMaxNumIterations;
     }
 
-    int GetMaxNumIterations()const
-    {
-    	return mMaxNumIterations;
-    }
-
-    void SetNewmarkBeta(double rBeta)
-    {
-    	mBeta = rBeta;
-    }
-
-    double GetNewmarkBeta()const
-    {
-    	return mBeta;
-    }
-
-    void SetNewmarkGamma(double rGamma)
-    {
-    	mGamma = rGamma;
-    }
-
-    double GetNewmarkGamma()const
-    {
-    	return mGamma;
-    }
-
-    bool GetUseLumpedMass()const
-    {
-    	return mUseLumpedMass;
-    }
-
-    void SetUseLumpedMass(bool rUseLumpedMass)
-    {
-    	mUseLumpedMass = rUseLumpedMass;
```

Comment:
This lumped feature is used in the Runge-Kutta explicit schemes

    StructureOutputBlockMatrix hessian2 = mStructure->BuildGlobalHessian2Lumped();